aicoe/sesheta/actions/pull_request.py

```diff
@@ -266,8 +291,8 @@
     return False
 
 
-async def needs_rebase(pull_request: dict = None) -> bool:
```

Comment:
why is this renamed?

Reply:
@goern as the function is checking if the PR is merged or mergeable. That's can be used for multiple things , why use it for only rebase. adding label would need a check if pr is merged or mergeable. so changed it to use it many other places without any issue.

Reply:
well, it specifically checks for the criteria of a merge, not for any label

Reply:
i understand it not checking for labels, however if a pr is merged there is not to added any label to it. so this function would be used other functions as well. 
Maybe it should be renamed to `check_merged` or `check_mergeable`!